assets/src/app/order/entity/entity.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, Input } from '@angular/core'
+import { Entity } from '../../models/nlp_entity'
+
+@Component({
+  selector: 'entity-component',
+  templateUrl: 'entity.component.html',
+  styleUrls: ['./entity.component.less'],
+})
+
+export class EntityComponent {
+
+  @Input() entities: Entity[];
+  @Input() word: string;
+  @Input() index: number;
+
+  constructor(
+  ) { }
+
+  ngOnInit() {
+  }
+
+  getStyle(index: number) {
```

Comment:
Is it possible to use less file to store entire style.
You can use class for that.

Reply:
i need variable in my style